evap/staff/importers.py

```diff
@@ -271,10 +271,13 @@ def read_one_enrollment(self, data):
     def process_course(self, course_data, sheet, row):
         course_id = course_data.name_en
         if course_id not in self.courses:
-            self.courses[course_id] = course_data
+            if any(course.name_de == course_data.name_de for course in self.courses.values()):
```

Comment:
that's O(enrolments * courses), can you perhaps put the german names into a set?